py_gtfs_rt_ingestion/batch_files.py

```diff
@@ -12,6 +12,7 @@
 from py_gtfs_rt_ingestion import DEFAULT_S3_PREFIX
 from py_gtfs_rt_ingestion import LambdaContext
 from py_gtfs_rt_ingestion import LambdaDict
+from py_gtfs_rt_ingestion import ProcessLogger
```

Comment:
new class thats used for the vast majority of logging. 

worth checking out before reading the rest here.

https://github.com/mbta/lamp/blob/lambda_logging_cleanup/py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/logging_utils.py